tinder_api.py

```diff
@@ -264,11 +264,18 @@ def match_info(match_id):
     except requests.exceptions.RequestException as e:
         print("Something went wrong. Could not get your match info:", e)
 
-def all_matches():
+# The web api calls this with count=60 and message=0, page_token will be None initially
+def all_matches(count=60, message=0, page_token=None):
     try:
-        url = config.host + '/v2/matches'
+        url = config.host + '/v2/matches?locale=en&count=' + str(count) + '&message=' + str(message) + '&is_tinder_u=false'
+        if page_token:
+            url = url + '&page_token=' + page_token
         r = requests.get(url, headers=headers)
-        return r.json()
+        json = r.json()
+        if 'data' in json.keys() and 'next_page_token' in json['data'].keys():
```

Comment:
```suggestion
        if 'data' in json and 'next_page_token' in json['data']:
```

Minor suggestion

Reply:
Can't believe I didn't know this. Need to work more on my python!